pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -266,6 +266,11 @@ func (m *kubeGenericRuntimeManager) getSeccompProfilePath(annotations map[string
 	return ""
 }
 
+// seccompRootDirectory returns the path for the localhost seccomp profiles.
+func (m *kubeGenericRuntimeManager) seccompRootDirectory() string {
```

Comment:
how frequently is this called? would it make sense to construct this once at startup? xref https://github.com/kubernetes/kubernetes/pull/16384 for cost of invoking Join methods repeatedly.

Reply:
Yes, I moved the construction of this path to the kuberuntime initialization: https://github.com/kubernetes/kubernetes/pull/103941/files#diff-0e1a8d22b0f1dc42d412749c6a916b4c76394ead739d4bfbf26564faedd74a2fR210